lib/application.ex

```diff
@@ -187,6 +188,7 @@ defmodule Commanded.Application do
         config
       end
 
+      @spec child_spec(opts :: Commanded.options()) :: Supervisor.child_spec()
```

Comment:
For consistency with the `start_link/1` callback function typespec:

```suggestion
      @spec child_spec(opts :: options()) :: Supervisor.child_spec()
```